openedx/core/djangoapps/video_config/models.py

```diff
@@ -1,11 +1,13 @@
 """
 Configuration models for Video XModule
 """
+from __future__ import absolute_import
 from django.db import models
 from django.db.models import BooleanField, TextField, PositiveIntegerField
 from config_models.models import ConfigurationModel
 from model_utils.models import TimeStampedModel
 from opaque_keys.edx.django.models import CourseKeyField
+import six
```

Comment:
minor nit: if you have a moment, could you run `isort` on this block? Not crucial, because we can do it in a later PR

Reply:
will do